### DISCUSSION TITLE
imagetk lib not installing

### DISCUSSION THREAD

**Discussion by saikiran994** [srixivas/PcapXray#63] [Q&A] [answered]
tried this command but no use
apt install python3-pil python3-pil.imagetk

dont know what to do and how to resolve it
python version is 3.7.5

**Comment by srixivas** [comment]
Could you elaborate on the OS details and any error message that you hit during installation/tool run?

**Ref:**
ImageTk dependency installation error could be resolved [like](https://stackoverflow.com/questions/48317606/python-cannot-import-imagetk/51229664).

**Comment by saikiran994** [comment]
kali linux 2020.4

import error for imagetk

i have tried "apt install python3-pil python3-pil.imagetk"

but its not working. every lib got executed but pil.imagetk not installing.

**Comment by saikiran994** [comment]
![mscanner error](https://user-images.githubusercontent.com/37449869/105478176-dca5cc80-5cc8-11eb-8134-b13b77a3cee3.PNG)

this is the error which i am facing right now. now able to use it in kali linux latest version

**Comment by srixivas** [comment, marked as answer]
Thanks for spotting this. I can also reproduce this dependency error in kali linux. 

Thanks to maintainers at https://github.com/home-assistant/core/issues/45918 and https://github.com/secdev/scapy/issues/3100. This has been fixed in v2.4.5.

Upgrade scapy to fix with py3.9
```
pip3 install scapy==2.4.5
```
